### PR TITLE
Fix for StartMinimized to wait for WebView2 initialization

### PR DESCRIPTION
**Pull request by Novack**
WebView2 was not properly loading if the main window was minimized on start. So the session actually started after the first time the window was restored.

### CODE REVIEW

File: skiffWindowsApp/Skiff Desktop/MainWindow.xaml.cs [hunk 1]
```diff
@@ -158,6 +155,7 @@ private async Task InitializeBrowser()
 
         private async void WebView2_CoreWebView2InitializationCompleted(object sender, CoreWebView2InitializationCompletedEventArgs e)
         {
+            ApplyWindowPreferences();
```
Comment (riverar) [inline skiffWindowsApp/Skiff Desktop/MainWindow.xaml.cs:158]: If WebView2 takes a while to initialize/fire that event, the window may jump around unexpectedly.

Should we maybe separate the window placement from any webview preferences to get the best of both worlds here?

Comment (Novack) [reply skiffWindowsApp/Skiff Desktop/MainWindow.xaml.cs:158]: Would you elaborate what you mean by jumping around? To give some context, the applied window preference is to start minimized (if that is enabled). The practical effect of this change, is that on Windows start the skiff window will be visible briefly, then minimize (if the user does not have the setting to minimize on start, wont see any difference whatsoever).

But Im interested on what you mean. Do you see a better path here? For reference, my approach was to take the simplest/shortest path: wait for initialization, then minimize.

Comment (riverar) [reply skiffWindowsApp/Skiff Desktop/MainWindow.xaml.cs:158]: I thought `RestoreWindow` was in the path of `ApplyWindowPreferences` for some reason, but it's not upon closer inspection. So looks good to me!

**Review by riverar — APPROVED** [review]

**Review by amilich — APPROVED** [review]
lgtm too - will release in the next couple days

**Pull request merged**